pkg/grpc/api.go

```diff
@@ -180,21 +179,21 @@ func (a *api) CallActor(ctx context.Context, in *internalv1pb.InternalInvokeRequ
 
 func (a *api) PublishEvent(ctx context.Context, in *runtimev1pb.PublishEventRequest) (*empty.Empty, error) {
 	if a.publishFn == nil {
-		err := errors.New("ERR_PUBSUB_NOT_FOUND")
+		err := status.Error(codes.Unavailable, "pubsub function is not configured")
```

Comment:
Please follow this spec - https://cloud.google.com/apis/design/errors#handling_errors

I think `FAILED_PRECONDITION` makes more sense than Unavaiable. 

FAILED_PRECONDITION, "no pubsub is configured"

Reply:
done.